types/three/src/loaders/Loader.d.ts

```diff
@@ -0,0 +1,47 @@
+import { LoadingManager } from './LoadingManager';
+
+/**
+ * Base class for implementing loaders.
+ */
+export class Loader {
+    constructor(manager?: LoadingManager);
+
+    /**
+     * @default 'anonymous'
+     */
+    crossOrigin: string;
+
+    /**
+     * @default: false
+     */
+    withCredentials: boolean;
+
+    /**
+     * @default ''
+     */
+    path: string;
+
+    /**
+     * @default ''
+     */
+    resourcePath: string;
+    manager: LoadingManager;
+
+    /**
+     * @default {}
+     */
+    requestHeader: { [header: string]: string };
+
+    /*
+	load(): void;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/three/src/loaders/Loader.d.ts;
  Line: 36;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.